src/Block/BannerBlock.php

```diff
@@ -47,6 +47,19 @@ public function getCMSFields()
         return parent::getCMSFields();
     }
 
+    /**
+     * For the backend, e.g. $block->CallToActionLink, return the database column which is
+     * a json encoded string.  If there is old data where null was mistakenly saved as a string 'null'
+     * then convert that to null
+     *
+     * @return string|null
+     */
+    public function getCallToActionLink()
+    {
+        $callToActionLink = $this->getField('CallToActionLink');
+        return $callToActionLink === 'null' ? null: $callToActionLink;
```

Comment:
Add a space after the `:` in the ternary?

Reply:
Yyes ternary, surprised phpcs didn't pick it up.  Have fixed.